pipeline.yaml

```diff
@@ -38,8 +46,51 @@ jobs:
     displayName: 'Export Keyvault Variables'
     name: exportKeyvault
 
+- job: MonitoringDeploy
+  displayName: Deploy Monitoring Infra
+  pool:
+    vmImage: 'Ubuntu 16.04'
+    #name: hmcts-agent-pool #TODO: change to hosted when merged
+  variables:
+    - group: aks-variable-group
```

Comment:
I see this is the slack URL - get this from keyvault instead.  Actually you can even link variable groups to keyvault - maybe that would save us having to have an extra 'Keyvault' job....